src/driver-marsgaming/marsgaming-probe.c

```diff
@@ -0,0 +1,150 @@
+#include "marsgaming-probe.h"
+
+#include "marsgaming-definitions.h"
+#include "marsgaming-query.h"
+
+static void
+marsgaming_probe_button_action(struct ratbag_button *button,
+			       const struct marsgaming_button_info *button_info)
+{
+	struct ratbag_button_action action = marsgaming_parse_button_to_action(button, button_info);
+	ratbag_button_set_action(button, &action);
+}
+
+static void
+marsgaming_probe_profile_leds(struct ratbag_profile *profile)
+{
+	struct marsgaming_profile_drv_data *profile_data = marsgaming_profile_get_drv_data(profile);
+	profile_data->led_report = marsgaming_query_profile_led(profile);
+	struct ratbag_led *led;
+	ratbag_profile_for_each_led(profile, led) {
+		led->type = RATBAG_LED_TYPE_LOGO;
```

Comment:
Remove this line, libratbag doesn't have LED types anymore and it causes a compilation error.
```suggestion
```